tools/package-unix.zsh

```diff
@@ -5,13 +5,15 @@ DIR="$(cd "$(dirname "${(%):-%N}")"/.. >/dev/null 2>&1 && pwd)"
 [ -z "$LLVM_DIR" ] && echo "Must set specific LLVM_DIR for packaging" && exit
```

Comment:
...It's really weird to have both `package-unix.zsh` and `package-unix.sh`.

Reply:
(to be more specific: it would be vastly preferable to have a single .sh that will work under both zsh and bash.)

Reply:
This works on my mac for both bash and zsh:

`DIR=$(cd -P -- "$(dirname -- "$0")" && printf '%s\n' "$(pwd -P)/")`

Reply:
The `#!/bin/bash` up top renders that moot. But I guess Apple hasn't deleted bash from their OS _yet_. We can just drop the zsh script.

Reply:
In that case, we should probably nuke Unix.cmake too and just toll it into the unix shell script. (Sorry for all the change requests after approval...)